src/index.js

```diff
@@ -91,8 +91,12 @@ const generateEDL = ({
 	fileNames,
 	sampleRate,
 	frameRate,
-	projectName
+	projectName,
+	adlUid,
```

Comment:
kinda hard to read the ADL UID in variable form.

Reply:
you mean `ADLUid` is more readable?

Reply:
leaving this for now